core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -134,6 +138,14 @@ public final String getPath() {
     return path;
   }
 
+
+  /**
+   * Set MetricsContext labels.
+   */
+  protected void setupMetricsContext(T appConfig) {
```

Comment:
Do we want to allow people to override this method?

Reply:
Yes to allow application to add additional labels to the MetadataContext. I suppose we could also just append the labels we want to the Configdef but it felt akward since they aren't really meant to be configurable.

Reply:
removed